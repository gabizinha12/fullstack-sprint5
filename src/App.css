.hide {
  display: none;
}

.alert {
  background-color: #f8d7da;
  border: 1px solid #f5c6cb;
  border-radius: 4px;
  color: #721c24;
  font-family: "Open Sans", sans-serif;
  font-weight: 600;
  padding: 10px;
  position: fixed;
  right: 10px;
  top: 10px;
  z-index: 9999;
}

.header {
  width: 100%;
  padding-bottom: 10px;
  border-bottom: 1px solid #e0e0e0;
}
@media (max-width: 1200px) {
  .header--floating {
    position: fixed;
    top: 0;
    background-color: #fff;
  }
}
.header__container {
  margin: auto;
  width: 95%;
  max-width: 1200px;
}
@media (max-width: 1200px) {
  .header__desktop {
    display: none;
  }
}
.header__mobile {
  display: none;
  margin-bottom: 25px;
}
@media (max-width: 1200px) {
  .header__mobile {
    display: flex;
    align-items: center;
  }
}
.header__logo {
  height: 70px;
  line-height: 70px;
  text-align: center;
}
@media (max-width: 1200px) {
  .header__logo {
    flex-grow: 1;
  }
}
.header__img {
  height: 25px;
  transform: translateY(10px);
}
.header__search {
  position: relative;
}
.header__icon {
  width: 24px;
  opacity: 26%;
  top: 8px;
  position: absolute;
  padding-left: 10px;
  color: rgba(0, 0, 0, 0.26);
  line-height: 40px;
}
.header__input {
  height: 40px;
  width: 100%;
  border-radius: 4px;
  border: 1px solid rgba(0, 0, 0, 0.23);
  padding-left: 40px;
}
.header__input:hover {
  border-color: #000;
}
.header__input::placeholder {
  font-size: 0.9375rem;
  font-weight: 600;
  font-family: "Open Sans";
  color: #9b9b9b;
}
.header__drawer {
  width: 24px;
  opacity: 54%;
  display: none;
}
.header__drawer--active {
  display: block;
}
.header__drawer .menu__title,
.header__drawer .close__title {
  text-transform: lowercase;
  font-size: 0.5625rem;
  font-weight: 400;
  line-height: 0;
}
@media (max-width: 1200px) {
  .header__menu {
    display: none;
  }
}
.header__menu .menu__list {
  display: flex;
  justify-content: space-between;
  margin-top: 10px;
  margin-bottom: 10px;
}
.header__menu .menu__link {
  font-size: 0.8125rem;
  font-weight: 600;
  color: #000;
  text-transform: uppercase;
  padding-bottom: 5px;
  vertical-align: baseline;
}
.header__menu .menu__link:hover {
  border-bottom: 2px solid #000;
}

.main {
  margin: 0 auto;
  width: 80%;
  max-width: 1100px;
  padding: 16px;
}
.main__breadcrumbs {
  margin-bottom: 16px;
}
.main__breadcrumbs .breadcrumbs__list {
  display: flex;
}
@media (max-width: 1200px) {
  .main__breadcrumbs .breadcrumbs__list {
    justify-content: center;
  }
}
.main__breadcrumbs .breadcrumbs__item {
  font-size: 0.8125rem;
  font-weight: 600;
  color: #585858;
}
@media (max-width: 1200px) {
  .main__breadcrumbs .breadcrumbs__item {
    display: none;
  }
}
.main__breadcrumbs .breadcrumbs__item:not(:last-child):hover {
  text-decoration: underline;
}
.main__breadcrumbs .breadcrumbs__item--active {
  color: #261e1e;
}
@media (max-width: 1200px) {
  .main__breadcrumbs .breadcrumbs__item--active {
    display: block;
    text-align: center;
    font-size: 1.4375rem;
    font-weight: 400;
    text-transform: uppercase;
  }
}
.main__breadcrumbs .breadcrumbs__separator {
  margin-left: 8px;
  margin-right: 8px;
  user-select: none;
}
.main__filters {
  border-top: 1px solid #e3e3e3;
  border-bottom: 1px solid #e3e3e3;
}
@media (max-width: 1200px) {
  .main__filters {
    display: none;
  }
}
.main__filters .filters__list {
  display: flex;
}
.main__filters .filters__item {
  display: flex;
  align-items: center;
  padding: 12px 14px;
}
.main__filters .filters__item:hover {
  cursor: pointer;
  background-color: rgba(38, 30, 30, 0.04);
}
.main__filters .filters__label {
  text-transform: uppercase;
  font-size: 0.6875rem;
  font-weight: 400;
  color: #261e1e;
}
.main__filters .filters__img {
  height: 20px;
  margin-left: 8px;
}
.main__products {
  margin-top: 20px;
  margin-bottom: 20px;
}
.main__products .products__row {
  margin: 4px;
}
.main__products .products__list {
  display: flex;
  justify-content: space-between;
  flex-wrap: wrap;
}
@media (max-width: 1200px) {
  .main__products .products__list {
    flex-wrap: wrap;
  }
}
@media (max-width: 1200px) {
  .main__products .products__card {
    flex-basis: 50%;
  }
}
.main__products .products__card .card {
  cursor: pointer;
  border: 1px solid white;
  padding: 8px;
  max-width: 230px;
}
.main__products .products__card .card:hover {
  box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
    0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 1px 3px 0px rgba(0, 0, 0, 0.12);
}
.main__products .products__card .card__img {
  max-width: 100%;
  max-height: 100%;
}
.main__products .products__card .card__description {
  font-size: 0.6875rem;
  font-weight: 400;
  color: rgba(0, 0, 0, 0.87);
  text-transform: uppercase;
  text-align: center;
  margin-top: 8px;
  margin-bottom: 10px;
}
.main__products .products__card .card__price {
  font-size: 0.8125rem;
  font-weight: 600;
  text-transform: uppercase;
  text-align: center;
}

.footer {
  border-top: 2px solid lightgray;
  padding-top: 40px;
  width: 100%;
}
.footer__container {
  margin: 0 auto;
  width: 90%;
}
.footer__contact .contact {
  display: flex;
  justify-content: space-between;
  align-items: center;
}
.footer__contact .contact__info {
  height: 50px;
  line-height: 50px;
  text-transform: uppercase;
  font-size: 0.8125rem;
}
.footer__contact .contact__input {
  height: 24px;
  line-height: 24px;
  border: 0px;
  border-bottom: 1px solid rgba(38, 30, 30, 0.42);
  outline: none;
  width: 300px;
  font-size: 0.9375rem;
  font-weight: 600;
  color: #000;
}
.footer__contact .contact__input:hover {
  border-bottom: 2px solid #261e1e;
}
.footer__contact .contact__input:focus-within {
  border-bottom: 2px solid #261e1e;
}
.footer__contact .contact__input::placeholder {
  font-style: italic;
  opacity: 42%;
  font-weight: 600;
}
.footer__contact .contact__btn {
  margin-left: 20px;
  color: rgba(0, 0, 0, 0.26);
  box-shadow: none;
  background-color: rgba(0, 0, 0, 0.12);
  border-radius: 4px;
  border: none;
  padding: 10px 20px 10px 20px;
}
.footer__contact .contact__btn__txt {
  font-size: 0.8125rem;
  text-transform: uppercase;
}
.footer__contact .contact__msg-app {
  font-size: 0.9375rem;
  height: 50px;
  line-height: 50px;
  text-transform: uppercase;
}
.footer__contact .contact .apps__img {
  width: 110px;
  margin-left: 8px;
}
.footer__links {
  background-color: #f6f6f6;
  margin-top: 40px;
}
.footer__links .links {
  display: flex;
}
.footer__links .links__contacts {
  width: 30%;
  display: flex;
}
.footer__links .links__contacts .col {
  flex-grow: 1;
  display: flex;
  justify-content: space-around;
  flex-direction: column;
}
.footer__links .links__contacts .contact-icon {
  text-align: center;
}
.footer__links .links__contacts .contact-icon__img {
  display: block;
  margin: auto;
  width: 25px;
}
.footer__links .links__contacts .contact-icon__text {
  padding: 8px;
  font-size: 0.8125rem;
  white-space: pre-line;
}
.footer__links .links__separator {
  width: 1px;
  border: none;
  background-color: rgba(0, 0, 0, 0.12);
}
.footer__links .links__companies {
  display: flex;
  width: 60%;
  padding: 50px;
}
.footer__links .links__companies .company-links {
  flex-grow: 1;
  margin-left: 8px;
  margin-right: 8px;
}
.footer__links .links__companies .company-links__item {
  padding-top: 8px;
}
.footer__links .links__companies .company-links__title {
  text-transform: uppercase;
  font-size: 0.875rem;
}
.footer__links .links__companies .company-links__text {
  line-height: 25px;
  font-size: 0.6875rem;
  font-weight: 400;
  color: black;
}
.footer__links .links__companies .company-links__text:hover {
  text-decoration: underline;
}
.footer__payment {
  margin-top: 25px;
  margin-bottom: 25px;
}
.footer__payment .payment {
  display: flex;
}
.footer__payment .payment__title {
  text-transform: uppercase;
  margin-bottom: 10px;
}
.footer__payment .payment__alternatives {
  width: 50%;
}
.footer__payment .payment__img {
  margin: 2px;
  width: auto;
}
.footer__payment .payment__social {
  flex-grow: 1;
}
.footer__payment .payment__social .social__icons {
  display: flex;
}
.footer__payment .payment__social .social__icon {
  width: 50px;
}
.footer__payment .payment__social .social__icon .icon__img {
  display: block;
  margin: auto;
}
.footer__payment .payment__social .social__icon .icon__text {
  padding-top: 4px;
  font-size: 0.6875rem;
  font-weight: 300;
  text-align: center;
}
.footer__payment .payment__accessibility {
  flex-grow: 1;
}
.footer__bottom {
  background-color: #f6f6f6;
  padding: 20px 0;
}
.footer__bottom .bottom {
  display: flex;
}
.footer__bottom .copyright {
  width: 83.333333%;
}
.footer__bottom .copyright__text {
  font-size: 0.5625rem;
  font-weight: 400;
  color: #261e1e;
}
.footer__bottom .copyright__stamp {
  margin: 0 auto;
}

/*# sourceMappingURL=styles.css.map */
