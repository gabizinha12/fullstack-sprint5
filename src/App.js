import React from "react";
import "./App.css";
import Footer from "./components/footer/Footer";
import Header from "./components/header/Header";
import { MessageProvider } from "./contexts/MessageContext";
import { CatgoriasContext } from "./contexts/CategoriasContext";
import { ProdutosContext } from "./contexts/ProductsContext";
import { LoadContext } from "./contexts/LoadingContext";

import Message from "./components/message/Message";
import Products from "./components/products/products";
import Loading from "./components/loading/loading";
function App() {
  return (
    <div className="App">
      <ProdutosContext>
        <LoadContext>
          <Loading />
          <CatgoriasContext>
            <MessageProvider>
              <Message />
              <Header></Header>
              <Products></Products>
              <Footer></Footer>
            </MessageProvider>
          </CatgoriasContext>
        </LoadContext>
      </ProdutosContext>
    </div>
  );
}

export default App;
