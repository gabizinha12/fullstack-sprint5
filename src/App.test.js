import React from "react";
import { render, screen } from "@testing-library/react";
import App from "./App";
describe("componente do header", () => {
  it("o snapshot deve permanecer o mesmo", () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
