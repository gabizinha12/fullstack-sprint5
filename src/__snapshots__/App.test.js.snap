// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`componente do header o snapshot deve permanecer o mesmo 1`] = `
<div
  class="App"
>
  <div
    class="loading "
  >
    <div
      class="bar bar1"
    />
    <div
      class="bar bar2"
    />
    <div
      class="bar bar3"
    />
    <div
      class="bar bar4"
    />
    <div
      class="bar bar5"
    />
    <div
      class="bar bar6"
    />
    <div
      class="bar bar7"
    />
    <div
      class="bar bar8"
    />
  </div>
  <section
    class="sc-hBMUJo dgUgZY"
  >
    <h1>
      Produtos carregados com sucesso
    </h1>
  </section>
  <header
    class="sc-bdnxRM ODqTo"
  >
    <div
      class="sc-gtsrHT Dgrcd"
    >
      <div
        class="sc-dlnjwi cosUdp"
      >
        <div
          class="sc-iCoGMd zqxxU"
        >
          <img
            alt="menu"
            class="sc-fujyAs"
            src="menu.svg"
          />
          <p
            class="sc-pNWdM lCbqY"
          >
            menu
          </p>
        </div>
        <div
          class="sc-iCoGMd kpOEcc"
        >
          <img
            alt="fechar menu"
            class="sc-fujyAs"
            src="close.svg"
          />
          <p
            class="sc-pNWdM lCbqY"
          >
            fechar
          </p>
        </div>
        <h1
          class="sc-jSFjdj bwmfWu"
        >
          <img
            alt="Logo Riachuelo"
            class="sc-gKAaRy gdxmWT"
            src="rchlo.svg"
          />
        </h1>
      </div>
      <div
        class="sc-hKFxyN jXZoXn"
      >
        <h1
          class="sc-jSFjdj bwmfWu"
        >
          <img
            alt="Logo Riachuelo"
            class="sc-gKAaRy gdxmWT"
            src="riachuelo.svg"
          />
        </h1>
      </div>
      <div
        class="sc-eCApnc gDvtbK"
      >
        <img
          alt="lupa"
          class="sc-jrsJWt iOvsrw"
          src="search.svg"
        />
        <input
          class="sc-kEqXSa gcoDLL"
          placeholder="O que você está procurando?"
          type="search"
        />
      </div>
      <nav
        class="sc-bqGGPW gFLTgU"
      >
        <ul
          class="sc-ksluID dYQQAW"
        />
      </nav>
    </div>
  </header>
  <main
    class="main"
  >
    <section
      class="sc-fKgJPI xPGIn"
    >
      <ol
        class="sc-bCwfaz jqymxT"
      />
    </section>
    <section
      class="sc-iJCRrE dUVYgl"
    >
      <ul
        class="sc-giAqHp hoOZDa"
      />
    </section>
    <section
      class="sc-ezzafa iLpKgu"
    >
      <ol
        class="sc-bYwzuL ipZSLr"
      />
    </section>
  </main>
  <footer
    class="footer"
  >
    <section
      class="footer__contact"
    >
      <div
        class="footer__container contact"
      >
        <p
          class="contact__info"
        >
          Receba ofertas e novidades por e-mail
        </p>
        <form
          action="/"
        >
          <input
            class="contact__input"
            placeholder="Digite aqui seu e-mail"
            type="email"
          />
          <button
            class="contact__btn"
            type="submit"
          >
            <span
              class="contact__btn__txt"
            >
              Cadastrar
            </span>
          </button>
        </form>
        <p
          class="contact__msg-app"
        >
          Baixe nosso App
        </p>
        <div
          class="apps"
        >
          <a
            href="/"
          >
            <img
              alt="Google Play"
              class="apps__img"
              src="google-play.webp"
            />
          </a>
          <a
            href="/"
          >
            <img
              alt="App Store"
              class="apps__img"
              src="app-store.webp"
            />
          </a>
        </div>
      </div>
    </section>
    <section
      class="footer__links links"
    >
      <div
        class="footer__container links"
      >
        <div
          class="links__contacts"
        >
          <div
            class="col"
          >
            <a
              class="contact-icon"
              href="/"
            >
              <img
                alt="Atendimento"
                class="contact-icon__img"
                src="ico-attendance.svg"
              />
              <p
                class="contact-icon__text"
              >
                Atendimento
              </p>
            </a>
            <a
              class="contact-icon"
              href="/"
            >
              <img
                alt="Perguntas Frequentes"
                class="contact-icon__img"
                src="ico-faq.svg"
              />
              <p
                class="contact-icon__text"
              >
                Perguntas Frequentes
              </p>
            </a>
          </div>
          <div
            class="col"
          >
            <a
              class="contact-icon"
              href="/"
            >
              <img
                alt="Fatura e Extrato"
                class="contact-icon__img"
                src="ico-invoice-and-statement.svg"
              />
              <p
                class="contact-icon__text"
              >
                Fatura e Extrato
              </p>
            </a>
            <a
              class="contact-icon"
              href="/"
            >
              <img
                alt="Trabalhe Conosco"
                class="contact-icon__img"
                src="ico-work-with-us.svg"
              />
              <p
                class="contact-icon__text"
              >
                Trabalhe Conosco
              </p>
            </a>
          </div>
        </div>
        <hr
          class="links__separator"
        />
        <ul
          class="links__companies company-links"
        >
          <li
            class="company-links"
          >
            <h3
              class="company-links__title"
            >
              Cartão Riachuelo
            </h3>
            <ul>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Midway Financeira
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Saiba como adquirir
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Veja sua fatura e extrato
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Buscando quitar suas dívidas?
                </a>
              </li>
            </ul>
          </li>
          <li
            class="company-links"
          >
            <h3
              class="company-links__title"
            >
              Sobre a Riachuelo
            </h3>
            <ul>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  A Empresa
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Portal de Fornecedores
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Relação com Investidores
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Assessoria de Imprensa
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Trabalhe Conosco
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Programa Jovem Aprendiz
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Cadastro de Startups
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                   
                  Mapa do Site
                   
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                   
                  Guide Shop
                   
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Venda na Riachuelo Marketplace
                </a>
              </li>
            </ul>
          </li>
          <li
            class="company-links"
          >
            <h3
              class="company-links__title"
            >
              Moda que Transforma
            </h3>
            <ul>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Na prática
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Entre Costuras
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Jeans Mais Transparente
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Ações Covid-19
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Free Free
                </a>
              </li>
            </ul>
          </li>
          <li
            class="company-links"
          >
            <h3
              class="company-links__title"
            >
              Ajuda
            </h3>
            <ul>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Atendimento
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Perguntas Frequentes
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Trocas e Devoluções
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Portal da Privacidade
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Política de Privacidade
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Termos e Condições de Uso
                </a>
              </li>
              <li
                class="company-links__item"
              >
                <a
                  class="company-links__text"
                  href="/"
                >
                  Regras e Resultados de Sorteios
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </div>
    </section>
    <section
      class="footer__payment"
    >
      <div
        class="footer__container payment"
      >
        <div
          class="payment__alternatives"
        >
          <h3
            class="payment__title"
          >
            Formas de Pagamento
          </h3>
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-rchlo1.webp"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-rchlo2.webp"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-rchlo3.webp"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-visa.png"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-master.webp"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-amex.webp"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-diners.png"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-cartao-elo.png"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-boleto.webp"
          />
          <img
            alt=""
            class="payment__img"
            src="ico-paypal.png"
          />
        </div>
        <div
          class="payment__social"
        >
          <h3
            class="payment__title"
          >
            Siga a Riachuelo
          </h3>
          <div
            class="social__icons"
          >
            <div
              class="social__icon icon"
            >
              <a
                href="/"
              >
                <img
                  alt=""
                  class="icon__img"
                  src="ico-facebook.svg"
                />
              </a>
            </div>
            <div
              class="social__icon icon"
            >
              <a
                href="/"
              >
                <img
                  alt=""
                  class="icon__img"
                  src="ico-instagram.svg"
                />
              </a>
              <p
                class="icon__text"
              >
                RCHLO
              </p>
            </div>
            <div
              class="social__icon icon"
            >
              <a
                href="/"
              >
                <img
                  alt=""
                  class="icon__img"
                  src="ico-instagram.svg"
                />
              </a>
              <p
                class="icon__text"
              >
                CASA RCHLO
              </p>
            </div>
            <div
              class="social__icon icon"
            >
              <a
                href="/"
              >
                <img
                  alt=""
                  class="icon__img"
                  src="ico-pinterest.svg"
                />
              </a>
            </div>
            <div
              class="social__icon icon"
            >
              <a
                href="/"
              >
                <img
                  alt=""
                  class="icon__img"
                  src="ico-twitter.svg"
                />
              </a>
            </div>
            <div
              class="social__icon icon"
            >
              <a
                href="/"
              >
                <img
                  alt=""
                  class="icon__img"
                  src="ico-youtube.svg"
                />
              </a>
            </div>
            <div
              class="social__icon icon"
            >
              <a
                href="/"
              >
                <img
                  alt=""
                  class="icon__img"
                  src="ico-linkedin.svg"
                />
              </a>
            </div>
          </div>
        </div>
        <div
          class="payment__accessibility"
        >
          <h3
            class="payment__title"
          >
            Acessibilidade
          </h3>
          <img
            alt=""
            src="ico-a11y.webp"
          />
        </div>
      </div>
    </section>
    <section
      class="footer__bottom"
    >
      <div
        class="footer__container bottom"
      >
        <div
          class="copyright"
        >
          <p
            class="copyright__text"
          >
            © Copyright 2017 - 2019. Todos os direitos reservados. Lojas Riachuelo S/A. Rua Landri Sales, 1.070, G02 Anexo B, Guarulhos - SP - Brasil
            <br />
            CEP 07250-130 CNPJ 33.200.056/0441-97 | IE 796.420.926.112 | SAC 4003-0515.Direção de fotografia de produtos por Adriano Adrião.
            <br />
            Pagamento com Cartão Riachuelo, parcelas mínimas de R$ 15,00. Para os demais cartões Visa, Mastercard, Elo, Diners, Hipercard e American Express, parcelas mínimas de R$ 30,00.
            <br />
            * COMPRE NO SITE e RETIRE NA LOJA com FRETE GRÁTIS, conforme lojas participantes. Consulte as lojas participantes nas opções de frete do seu pedido, conforme CEP de entrega cadastrado.
            <br />
            Frete único de R$ 4,99 para Sul e Sudeste em compras a partir de R$ 129,00. Demais regiões o valor estará sujeito a variações. Condição por tempo indeterminado, para tipo de entrega “normal”.
            <br />
            Preços, promoções, condições de pagamento, valores de frete e disponibilidade de estoque são aplicados somente nas compras efetuadas na loja online Riachuelo. Existe uma limitação de peças iguais no mesmo pedido que muda de acordo com o tipo de produto.
            <br />
            Em caso de divergência de valores entre a página do produto e a Sacola de Compras, a Riachuelo se reserva a garantir o valor exibido na Sacola de Compras.
          </p>
        </div>
        <div
          class="copyright__stamp"
        >
          <img
            alt=""
            src="ra1000.webp"
          />
        </div>
      </div>
    </section>
  </footer>
</div>
`;
